Week2/homework/maartjes-work.js

```diff
@@ -42,24 +67,35 @@ const tuesday = [
   },
 ];
 
-const maartjesTasks = monday.concat(tuesday);
-const maartjesHourlyRate = 20;
+const durations = monday.duration.concat(tuesday.duration);
+console.log(durations);
 
-function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
-}
+//Map the tasks to durations in hours.
 
-// eslint-disable-next-line no-unused-vars
-const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
+const hours = durations.map(function(duration) {
```

Comment:
At the moment `durations` is undefined, thus the argument `duration` is empty. Once you made the above code work try to log to the console what `duration` inside the callback is. Then proceed.